### PR TITLE
New command implementation

### PR DESCRIPTION
**Pull request by RohitValiveti**
### Summary

This PR:
- Addresses Jira ticket [CISLUNAR-154](https://cislunarsoftware.atlassian.net/browse/CISLUNAR-154)     
- Implements a new command to change the Earth/Sun/Moon threshold to help with testing the find_with_hough_transform_and_contours.findBody() opnav find algorithm

### Testing

- Tested with a unit test
- Find test in tests/command_test.py test_setEMSThresh()
- Tested uplinked data and downlinked

### Notes

- Looking through the other commands, one concern/thought I had was if this command may be redundant, given the existence of the set_param() command, which can change any single command. However, incorporating a single command for changing all of the parameters of only a particular find algorithm may be of use for testing

### Comments
- [x] Add an x between the brackets if you commented your code well!

### CODE REVIEW

**Review by tmf97 — APPROVED** [review]
Looks solid!
I see why this may feel a bit redundant, because the `set_param` command already exists. One functionality you could add to this command would to add some checks to the inputs before setting the parameter (i.e. is each parameter between 0 and 1? [or is it 0 to 255?]). This adds a safety net which prevents errors that could be cause by operator error or bit flips from radiation

File: tests/command_test.py [hunk 1]
```diff
@@ -48,6 +48,35 @@ def test_commands(self):
         self.assertEqual(downlink_command.id, CommandEnum.SetParam)
         self.assertEqual(downlink_args[ck.VALUE], new_value)
 
+    def test_setEMSThresh(self):
```
Comment (sjz38) [inline tests/command_test.py:51]: Add a comment here describing what the test is doing. See the other tests in this file for reference

**Comment by sjz38** [comment]
@tmf97 Did you want @RohitValiveti to implement the changes you suggested? I'm fine going ahead with the merge.

**Comment by RohitValiveti** [comment]
My bad, I forgot to add the check. I'll get on that tonight

**Comment by RohitValiveti** [comment]
@tmf97 @sjz38 Just added assert statements for the new threshold values to be between 0 and 1, lmk if you think its good

File: utils/constants.py [hunk 2]
```diff
@@ -137,6 +137,10 @@ class CommandKwargs(StringEnum):
     IGNORE = "ignore"
     PASSWORD = "password"
 
+    EARTH_THRESH = "earth threshold"
```
Comment (sjz38) [inline utils/constants.py:140]: change these to be snake-case (add underscore instead of space)

**Review by sjz38 — CHANGES_REQUESTED** [review]
see above

**Comment by RohitValiveti** [comment]
Just changed

**Review by tmf97 — APPROVED** [review]
Looks solid, feel free to Squash and Merge whenever. In the future let's lower the turnaround time on features like this

**Pull request merged**